### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by Sick-Carlito**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <a class="one" href="#">Mac</a>
+        <a class="one" href="#">iPod</a>
+        <a class="one" href="#">iPhone</a>
+        <a class="one" href="#">iPad</a>
+        <a class="one" href="#">iTunes</a>
+        <a class="one" href="#">Support</a>
+        <i class="fas fa-search"></i>
+        <input type="search" name="search">
+      </div>
+      <div class="verse">
```
Comment (Oghenebrume50) [inline index.html]: The content of this container are actually links and images, please check this out on the original site for comparison and implement according

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi @Sick-Carlito great start to this project, you are doing amazing :smiley: 
However, this is the current displace of the bottom part, it needs some fixing :wink: 

Please can you use more semantic HTML like <nav> for navbar and <footer> for the footer. Do these things and submit for a re-review :+1:

**Comment by Sick-Carlito** [comment]
The size is just fine on my size. I didn't manage to find what the problem was on your side. Thank you.
Screenshot from 2019-08-29 18-37-33

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Good Job @Sick-Carlito 

## The [clone](https://raw.githack.com/Sick-Carlito/Apple-Page/9b5bb21b0ba046b897b89d6ab4c7969e34e39f3a/index.html) looks okay to me... :ok: 

@Oghenebrume50 might be using a different percentage zoom in the browser. :thinking: 

There's still one comment raised by @Oghenebrume50 you haven't fixed, kindly **use semantic HTML elements**.
     > Remember to use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp).

Fix this and submit for another code review request. :blush: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Comment by samgaco** [comment]
### Everything looks correct :+1: The project is approved, congratulations on finishing your first solo project! :1st_place_medal: :clap: 

### You can go ahead and merge to the master branch and submit the project completion form. :100:

**Comment by Sick-Carlito** [comment]
Thanks Guys!!

**Pull request merged**